src/inject/inject.js

```diff
@@ -9,10 +9,10 @@ chrome.extension.sendMessage({}, function() {
       clearInterval(readyStateCheckInterval);
       chrome.runtime.onMessage.addListener(routeCommand);
     }
-  }, 10);
+  }, 3);
 
   function routeCommand(request) {
-    card = $('.active-card');
+    card = $('.card-detail-window');
```

Comment:
It's still there. It's the card that your mouse is currently hovering over. It's how the spacebar shortcut in Trello identified which card to assign to you.